kolibri/plugins/learn/assets/src/state/actions.js

```diff
@@ -62,6 +62,7 @@ function _contentState(data) {
     available: data.available,
     files: data.files,
     progress,
+    breadcrumbs: [],
```

Comment:
It was removed as part of @rtibbles performance optimization (we no longer pass ancestors along with every node).